README.md

```diff
@@ -51,7 +51,17 @@ After running `docker-compose up` successfully, the output in your terminal shou
 
 - [Python 3.10](https://github.com/okfde/fragdenstaat_de/blob/21faa6893a582a02f2a96f4ccce96fddab13bec6/requirements.txt#L2) (for the backend server)
 
-> Note: You can use [pyenv](https://github.com/pyenv/pyenv) to manage your Python versions.
+    > Note: You can use [pyenv](https://github.com/pyenv/pyenv) to manage your Python versions.
+
+- PostgreSQL `psql` command line client (for Django admin `dbshell` support)
+
+    > Note: Comes with a full PostgreSQL installation.
+
+- [GDAL (Geospatial Data Abstraction Library)](GDAL_LIBRARY_PATH) (GIS support for PostgreSQL)
```

Comment:
I typically do a `export GEOS_LIBRARY_PATH=/opt/homebrew/lib/libgeos_c.dylib` here as well - have you seen that before?

Reply:
I forgot that from the meeting the other night. I'll include that one, too.

In my case, I don't have to set either so I'm kinda guessing here; trying to be helpful.